xapian-letor/include/xapian-letor/ranker.h

```diff
@@ -117,9 +117,9 @@ class XAPIAN_VISIBILITY_DEFAULT Ranker : public Xapian::Internal::intrusive_base
      *  @exception FileNotFoundError will be thrown if file not found at supplied path
      *  @return A vector of docids after ranking.
      */
-    std::vector<Xapian::docid> rank(const Xapian::MSet & mset,
-				    const std::string & model_key = std::string(),
-				    const Xapian::FeatureList & flist = Xapian::FeatureList());
+    void rank(Xapian::MSet & mset,
+	      const std::string & model_key = std::string(),
+	      const Xapian::FeatureList & flist = Xapian::FeatureList());
```

Comment:
`@return` above is now incorrect.

Please check the diff against the branch point and fix any other instances of this too.